Instagram/Instagram/NotificationPage/NotificationModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  NotificationModel.swift
+//  Instagram
+//
+//  Created by Владислав Лымарь on 19.10.2022.
+//
+
+import Foundation
+
+///  Заполнение данными 
+struct NotificationModel {
```

Comment:
```suggestion
struct Notification {
```
Model не используем это слово и так понятно что это